app/src/main/res/values/strings.xml

```diff
@@ -603,6 +603,7 @@
     <string name="folderLocationTitle">Select Location</string>
     <string name="addFolder">Add Folder</string>
     <string name="editFolder">Edit Folder</string>
+    <string name="bookmarkFolderDeleteDialogTitle" instruction="Dialog title for deleting a folder with bookmarks">"Delete folder and its contents?"</string>
```

Comment:
You can put this new string resource in donottranslate.xml and then, in another PR move it to strings.xml to trigger translations